db/migrate/20211126041731_create_addresses.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAddresses < ActiveRecord::Migration[6.1]
+  def change
+    create_table :address do |t|
+      t.integer :accounts_id
```

Comment:
Sửa tất các các file tương tự em nhé, 
theo convention cho chuẩn hỉ.
VD: accounts - orders có quan hệ 1 - n thì: orders (account_id, ...)
```suggestion
      t.integer :account_id
```

Reply:
dạ chị ơi sửa là sửa trong file migrate , hay là sửa trong db luôn vậy chị 
![image](https://user-images.githubusercontent.com/56921227/143827297-1c274f92-9ddc-45df-8d3c-1048dbbf373d.png)

Reply:
Em chạy câu lệnh rails db:rollback VERSION để quay trở lại version muốn sửa.
Thự hiện sửa trong file migrate tương ứng 
chạy câu lệnh rails db:migrate để cập nhập thay đổi